### ISSUE TITLE
NUX tips: Close button tooltip partially hidden on some tips

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#7562]
When the "new user experience" tips are displayed, the tips Close button tooltip is partially hidden in some tips. To reproduce, for example in Chrome:

- go to the Add New post page
- open the Chrome dev tools > Application > Storage > Local storage, and delete the entry ~~"GUTENBERG_NUX_1" if any~~ Edit: the storage mechanism has changed, you need now to clear the whole Gutenberg storage entry
- refresh the page
- the first tip should appear
- tab to or mouse over the Close button
- the button tooltip appears and it's fully visible:

<img width="1278" alt="screen shot 2018-06-27 at 11 44 39" src="https://user-images.githubusercontent.com/1682452/41966985-d536fec2-7a00-11e8-8667-dc8133456753.png">

- click on "See next tip" to go to the following tips
- tab to or mouse over the Close button
- the tooltip is partially hidden behind the tip
- worth also noting the tooltip appears in the same place where the "pulsing dot" is

<img width="1280" alt="screen shot 2018-06-27 at 11 44 52" src="https://user-images.githubusercontent.com/1682452/41967121-2f2d2bc2-7a01-11e8-8beb-8cd42e9518e6.png">

<img width="1280" alt="screen shot 2018-06-27 at 11 45 05" src="https://user-images.githubusercontent.com/1682452/41967122-2f4b0d9a-7a01-11e8-8ea1-ffdd8c2299c2.png">

Worth noting the tooltip position is dynamically calculated to stay within the viewport, and that seems to work correctly. However, seems there's something wrong with the z-index. Not sure why when the popover takes the `is-bottom` class it gets a lower z-index.

```
.components-popover:not(.is-mobile).is-bottom {
    top: 100%;
    z-index: 99990;
}
```

However, I guess the whole position of the tip should be reviewed, because the tooltip position conflicts with the pulsing dot.

**Comment by afercia** [comment]
Still can reproduce on current master

**Comment by aduth** [comment]
Confirmed this is still an issue as of May 31, 2019.

Debugging:

It is caused by this style, specific to bottom popovers:

https://github.com/WordPress/gutenberg/blob/1d2d1fc9dc88c97cb8f23472b03cc6a2d8241df0/packages/components/src/popover/style.scss#L126

This was introduced in #5630 (cc @faishal, @karmatosed, @jorgefilipecosta). I don't think this style should have been introduced this way, since the fix considered for #5630 is very specific to a particular popover, and should not apply inherently to any bottom-oriented popover (evidenced by this bug).

**Comment by noisysocks** [comment]
#18041 has addressed this by removing dot tips in lieu of a welcome guide modal.
